.gitignore

```diff
@@ -30,6 +30,7 @@
 .depend
 /.config
 /.config.old
+/.config\ *
```

Comment:
why need this?

Reply:
I meant to delete this (and the ".context *" one).  I had a conflict on my end with iCloud and had just added these to clean up the git change list.